addon/templates/components/email.hbs

```diff
@@ -0,0 +1,27 @@
+{{#unless @required}}
+  <span class="cs-component-email--optional" data-test-cs-component-email-optional>
+    Optional
+  </span>
+{{/unless}}
+<input
+  type="email"
+  name="email"
+  id="{{this.elementId}}-input"
+  class="cs-component-email--input {{if @required "required"}}"
+  required={{@required}}
+  autocomplete="off"
```

Comment:
Should we also pass `@value` in here to take advantage of the `trim` done in the validation?

Reply:
Do you mean adding `value={{value}}`?

Reply:
Yes